index.html

```diff
@@ -230,5 +230,79 @@ <h2>About Me</h2>
           </ul>
         </div>
     </section><!-- #About me -->
+      <!-- ================
+              contact-me
+             ================ -->
+             <section class="contact-me" id="contact-me">
+                <h6 id="text">
+                    I'm always interested in hearing about new projects,
+                    so if you'd like to chat please get in touch.
+                </h6>
+                <form id="form" action="https://formspree.io/f/moqrjalz" method="post">
+                  <input
+                    name="fullname"
+                    id="fullname"
+                    class="stored"
+                    type="text"
+                    placeholder="Full name"
+                    minlength="5"
```

Comment:
[OPTIONAL] It's better to remove `minlength` since some people might have a shorter name e.g. Bob